Tribler/community/multichain/community.py

```diff
@@ -282,6 +283,75 @@ def get_statistics(self, public_key=None):
             statistics["total_down"] = 0
         return statistics
 
+    @blocking_call_on_reactor_thread
+    def get_nodes(self, public_key=None, neighbor_radius=1):
+        """
+        Returns a dictionary with the neighboring nodes of a certain focus node within a certain radius,
+            regarding the local multichain database
+
+        :param public_key: the public key of the focus node
+        :param neighbor_radius: the radius within which the neighbors have to be returned
+        :return: a dictionary with nodes
+        """
+        if public_key is None:
+            public_key = self.my_member.public_key
+        # TODO: make call instead of using dummy data
+        #list_of_nodes = self.persistence.get_list_of_nodes(public_key, neighbor_radius)
```

Comment:
The commented code is the future code in this case: when #45 is ready, this will be uncommented and the dummy data removed.